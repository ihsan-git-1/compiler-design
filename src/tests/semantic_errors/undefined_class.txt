class test {

    int a(){
            int b =3;
            double b =2;
            while(a != b){
                if(num > dNumber){

                    int count = 3;

                }
            }
    }
}