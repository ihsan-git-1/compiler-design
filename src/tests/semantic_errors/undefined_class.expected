The name b already defined (5,20)
Undefined name a at (6,19)
Undefined name num at (7,20)
Undefined name dNumber at (7,26)
